app/views/devise/sessions/_mfa_enrolment.erb

```diff
@@ -2,28 +2,32 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-one-half">
-
-      <%= f.label :totp_code, class: "govuk-label" %>
-      <%= f.text_field :totp_code, autofocus: true, autocomplete: 'off', class: "govuk-input govuk-input--width-10" %>
-      <%= f.hidden_field :email, value: { email: false } %>
-
+      <div class="govuk-!-margin-bottom-7">
+        <div class="mfa-qr-code">
+          <%= raw(@secret_code_svg) %>
+        </div>
+      </div>
   </div>
-
-  <div class="govuk-grid-column-one-half">
-    <div class="mfa-qr-code">
-      <%= raw(@secret_code_svg) %>
+  <div class="govuk-grid-column-full">
+    <details class="govuk-details" data-module="govuk-details">
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          <%= t('mfa_enrolment.no_qr') %>
+        </span>
+      </summary>
+      <div class="govuk-details__text">
+        <%= t('mfa_enrolment.no_qr_instructions') %>
+        <pre><%= @secret_code %></pre>
+      </div>
+    </details>
+    <div>
+      <div class="govuk-!-margin-bottom-7">
+        <%= f.label :totp_code, class: "govuk-label" %>
```

Comment:
the label should be as per the screenshot

Reply:
You're right. For some reason I thought it was just the layout that had to be aligned with the docs. Will fix now.